### DISCUSSION TITLE
Text shadows not possible with text-clipped gradients?

### DISCUSSION THREAD

**Discussion by enlivn** [chakra-ui/chakra-ui#8459] [Q&A] [answered]
If I apply a gradient to text _and_ apply a text shadow, it looks wrong (you can try it out in the chakra UI playground live example [here](https://v2.chakra-ui.com/docs/styled-system/gradient#text-gradient-api)):

```
<Text
  bgGradient='linear(to-l, #7928CA, #FF0080)'
  bgClip='text'
  fontSize='6xl'
  fontWeight='extrabold'
  textShadow="0px 1px 2px black"
>
  Welcome to Chakra UI
</Text>
```

this results in 

<img width="809" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/78372052/2decd66a-cc1c-4ffa-a798-5a3826992d8f">

I can fake it by adding another `<Text>` (that the user can't actually see) below this one:

```
<Stack>
<Text
  position="absolute"
  bgGradient='linear(to-l, #7928CA, #FF0080)'
  bgClip='text'
  fontSize='6xl'
  fontWeight='extrabold'
>
  Welcome to Chakra UI
</Text>
<Text
  fontSize='6xl'
  fontWeight='extrabold'
  textShadow="0px 1px 2px black"
>
  Welcome to Chakra UI
</Text>
</Stack>
```

which gives:
<img width="805" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/78372052/c7945368-7193-4f43-a900-ea6ae47112c9">

If this is indeed a limitation, it would be nice to add a note to that section. I'd be happy to PR that in.

**Comment by segunadebayo** [comment, marked as answer]
This is CSS, not a Chakra bug. `background-clip: text` clips `text-shadow` with the fill.

Keep the gradient on one layer. Put the shadow on a second copy behind it, as you already did.
